### PR TITLE
[ADR] Enhance existing ADRs

### PR DESCRIPTION
**Pull request by chibenwa**
I have been doing a bit of archeology...

 - Update the status if needed
 - Indicate whether it had been implemented or not
 - Link the JIRA ticket (and thus the PRs as they should be references in the JIRA)
 - Link the pull request discussing the very ADR
 - If possible, link mailing list discussions

 Globally, we should try to link better sources of information together.

### CODE REVIEW

**Comment by jeantil** [comment]
thanks a lot for these updates !

File: src/adr/0039-distributed-blob-garbage-collector.md [hunk 1]
```diff
@@ -58,6 +60,10 @@ store the blob for. This can make some components harder to implement as we will
 Since we will not build a distributed task scheduler. To increment the `reference generation` and launch periodically a
 `garbage collection iteration`, the scheduling will be done by an external scheduler (cron job, kubernetes cronjob ...)
  which will call a webadmin endpoint to launch this task periodically.
+ 
+## References
+
+
```
Comment (Arsnael) [inline src/adr/0039-distributed-blob-garbage-collector.md:66]: empty references?

Comment (chibenwa) [reply src/adr/0039-distributed-blob-garbage-collector.md:66]: Likely I found nothing!

Maybe implied actors @remk @rouazana could be helping here?

File: src/adr/0026-removing-configured-additional-mailboxListeners.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ Date: 2020-04-03
 
 Accepted (lazy consensus)
 
+Not implemented yet.
```
[the remark this answers is not in the document]
Comment (chibenwa) [reply src/adr/0026-removing-configured-additional-mailboxListeners.md:9]: > is this comment really necessary?

Yes, better be more clear than not enough!

**Review by Arsnael — APPROVED** [review]

File: src/adr/0039-distributed-blob-garbage-collector.md (hunk 1 above)
Comment (rouazana) [reply src/adr/0039-distributed-blob-garbage-collector.md]: https://issues.apache.org/jira/browse/JAMES-3150 ?
And probably the PR creating this ADR?

Comment (chibenwa) [reply src/adr/0039-distributed-blob-garbage-collector.md]: > And probably the PR creating this ADR?

Which is? Sorry, I did not find it!

Comment (rouazana) [reply src/adr/0039-distributed-blob-garbage-collector.md]: https://github.com/linagora/james-project/pull/3122

probably a mistake to not have put it on the Apache repo

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**